models/Recording.ts

```diff
@@ -737,6 +752,26 @@ from (
   // INSTANCE METHODS
   //------------------
 
+  Recording.prototype.isInStation = function (station: Station | CreateStationData) {
+    if (station.hasOwnProperty('lat')) {
```

Comment:
Do we need this or should we always calculated for all stations and take minimum?